<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QWidget {
        background-color: rgb(63,63,63);
        color: rgb(197,197,197);
        font: 10pt "MS Shell Dlg 2";
    }
    QListWidget {
        background-color: rgb(50,50,50);
        border: none;
    }
    QListWidget::item {
        padding: 10px;
    }
    QListWidget::item:selected {
        background-color: rgb(85,85,85);
    }
    QTextEdit, QLineEdit {
        background-color: rgb(50,50,50);
        border: 1px solid rgb(90,90,90);
        border-radius: 6px;
    }
    QPushButton {
        background-color: rgb(136,136,136);
        color: black;
        border-radius: 6px;
        padding: 6px;
    }
    QPushButton:hover {
        background-color: rgb(160,160,160);
    }
   </string>
  </property>
  <widget class="QFrame" name="leftPanel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>260</width>
     <height>580</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <widget class="QLabel" name="labelUsers">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;span style="font-size:14pt; font-weight:600;"&gt;Users&lt;/span&gt;</string>
    </property>
   </widget>
   <widget class="QListWidget" name="userList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>240</width>
      <height>520</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="rightPanel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>810</width>
     <height>580</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <widget class="QLabel" name="chatTitle">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>400</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;span style="font-size:14pt; font-weight:600;"&gt;Select a user&lt;/span&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="chatArea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>770</width>
      <height>430</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="messageInput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>650</width>
      <height>40</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Type a message...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>500</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
